bindings/python/visualize/meshcat_visualizer.py

```diff
@@ -512,6 +539,16 @@ def load_box(self, x, y, z, shape_name="iDynTree", color=None):
             solid_shape=box, shape_name=shape_name, color=color
         )
 
+    def load_line(self, vertices, linewidth, shape_name="iDynTree", color=None):
```

Comment:
I realized we could have a more debuggable default shape name (like "idyntree-line" or similar), anyhow this is not a problem just for the line.